fastlane/lib/fastlane/actions/swiftlint.rb

```diff
@@ -76,6 +77,10 @@ def self.available_options
                                        is_string: false,
                                        default_value: :lint,
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :path,
+                                       description: "Specify path to lint",
+                                       is_string: true,
+                                       optional: true),
```

Comment:
Can we verify the path exists using `ConfigItem`'s `verify_block` argument?

```
verify_block: proc do |value|
    UI.user_error!("Couldn't find path '#{File.expand_path(value)}'") unless File.exist?(value)
end),
```

Reply:
Done.